shared/teams/new-team/wizard/create-channels.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Container from '../../../util/container'
+import * as Styles from '../../../styles'
+import {pluralize} from '../../../util/string'
+import {ModalTitle} from '../../common'
+
+type Props = {
+  teamname: string
+}
+
+const CreateChannel = (props: Props) => {
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+
+  const [channels, setChannels] = React.useState<Array<string>>(['hellos', 'random', ''])
+  const setChannel = (i: number) => (value: string) => {
+    channels[i] = value
+    setChannels([...channels])
+  }
+  const onClear = (i: number) => {
+    channels.splice(i, 1)
+    setChannels([...channels])
+  }
+  const onAdd = () => {
+    channels.push('')
+    setChannels([...channels])
+  }
+
+  const onBack = () => dispatch(nav.safeNavigateUpPayload())
+
+  const numChannels = channels.filter(c => !!c.trim()).length
+  const continueLabel = numChannels
+    ? `Continue with ${numChannels} ${pluralize('channel', numChannels)}`
+    : 'Continue without channels'
+
+  return (
+    <Kb.Modal
+      onClose={onBack}
+      header={{
+        leftButton: <Kb.Icon type="iconfont-arrow-left" onClick={onBack} />,
+        title: <ModalTitle teamname={props.teamname} title="Create channels" />,
+      }}
+      footer={{content: <Kb.Button fullWidth={true} label={continueLabel} />}}
+      allowOverflow={true}
+    >
+      <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.banner} centerChildren={true}>
+        <Kb.Icon type="icon-illustration-teams-channels-460-96" />
+      </Kb.Box2>
+      <Kb.Box2
+        direction="vertical"
+        fullWidth={true}
+        style={styles.body}
+        gap={Styles.isMobile ? 'xsmall' : 'tiny'}
+      >
+        <Kb.Text type="BodySmall">Channels can be joined by anyone in the team, unlike subteams.</Kb.Text>
+        <ChannelInput isGeneral={true} />
+        {channels.map((value, idx) => (
+          <ChannelInput key={idx} onChange={setChannel(idx)} value={value} onClear={() => onClear(idx)} />
+        ))}
+        <Kb.Button mode="Secondary" icon="iconfont-new" tooltip="" onClick={onAdd} />
```

Comment:
This is a little bit wider in the design. Also, any reason not to include a tooltip?

Reply:
I made it wider. I'm going to lobby to remove the tooltip requirement on Icon buttons